pkgs/top-level/all-packages.nix

```diff
@@ -25795,6 +25795,17 @@ in
 
   secretscanner = callPackage ../tools/security/secretscanner { };
 
+  semiphemeral = callPackage ../tools/social/twitter/semiphemeral {
+    inherit (python3Packages)
+    buildPythonApplication
+    fetchPypi
+    click
+    sqlalchemy
+    flask
+    tweepy
+    colorama;
```

Comment:
I had to remove these too! Otherwise you get the error 

```
error: anonymous function at /home/aj/proj/nix-nixos/nixpkgs/pkgs/tools/social/twitter/semiphemeral/default.nix:1:1 called with unexpected argument 'buildPythonApplication', at /home/aj/proj/nix-nixos/nixpkgs/lib/customisation.nix:69:16
```

Reply:
Yes, you do have to remove them.